web/src/utils/dateUtil.ts

```diff
@@ -37,19 +37,20 @@ export const getNowYesterdayInLong = (): number => {
  */
 interface DateTimeStyle {
   timezone: string;
-  use12hour: boolean | undefined;
+  time_format: 'browser' | '12hour' | '24hour';
```

Comment:
The frontend is using `hour` and the backend is using `hours`

Reply:
Geez my bad, do you have a preference for which one to use?

Reply:
Just did some quick googling, and I think `12hour` and `24hour` seem to be most common way to refer to time formats.

Reply:
Do you know a good way to do that? The enum variable name can't start with a digit, not sure if there is a workaround for that

Reply:
Nevermind looks like it works on both sides with just the string value